@kiva/kv-components/vue/.storybook/tailwind.css

```diff
@@ -1,3 +1,5 @@
+@import url('https://use.typekit.net/pmj7shs.css');
```

Comment:
Is there an alternative to importing a 3rd party CSS?

Reply:
Not for fonts thorugh adobe typekit. They must be included either as a direct css include in the html or imported like sot. It is our preferred font. Should we find any serious issues with this approach we may be able to switch to a different typeface from google fonts that does allow download.